api/src/main/java/org/ehcache/expiry/Expirations.java

```diff
@@ -57,11 +57,24 @@
     return new TimeToIdleExpiry(timeToIdle);
   }
 
+  /**
+   * Builder allowing to specify TTL for creation, access and update. Unspecified values will be set to {@code Duration.INFINITE}
+   * meaning not expiration.
+   *
+   * @return an {@link Expiry} builder
+   */
+  public static <K, V> ExpiryBuilder<K, V> builder() {
```

Comment:
The builder should be in the `builder` package or the `builder()` method as well? If the later, it will be strange for the users since other builder methods are already in `Expirations`